mods/ra/rules/structures.yaml

```diff
@@ -408,13 +408,13 @@ TSLA:
 		Name: Tesla Coil
 		Description: Advanced base defense.\nRequires power to operate.\nCan detect cloaked units.\n  Strong vs Vehicles, Infantry\n  Weak vs Aircraft
 	Building:
-		Footprint: _ x
-		Dimensions: 1,2
+		Footprint: x
```

Comment:
This is the default footprint, so you can remove it completely (here and the other defenses).